src/Copy/Text.elm

```diff
@@ -355,3 +358,8 @@ chompTrailingUrlSlash urlString =
 urlToDisplay : String -> String
 urlToDisplay url =
     Url.fromString url |> urlRecombiner |> chompTrailingUrlSlash
+
+
+googleMapSearchUrl : String -> String
```

Comment:
This could be a `Key` rather than an exposed utility function which might be more in keeping with the design of the module.

```
GoogleMapSearchUrl String
```